gcp/appengine/blog/layouts/posts/single.html

```diff
@@ -1,3 +1,6 @@
+.content {
```

Comment:
This should go into our styles.scss instead. @another-rex can you please help Hayley with this change?